packages/binding-mbus/tsconfighead.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
May I ask you to follow the same configuration of the other packages? Here's an example:
```json
{
	"extends": "../../tsconfig.json",
	"compilerOptions": {
		"outDir": "dist"
	},
	"include": [
		"src/**/*"
	]
}
```
It is a little bit inconvenient handling different tsc configurations per package